.dockerignore

```diff
@@ -1,5 +1,6 @@
 .idea
 .env*
+gcp-private-key.json

```

Comment:
Co to dat pod `var/gcp/`, jako davame treba MySQL certifikat. `var`/ je objecne ignorovany, neverzovany, urceny pro ruzny lokalni soubory nutny pro beh appky. Neco jako `var/gcp/service_account.json`

Reply:
nj, tady zrovna žádnej var není, ale přesunul jsem to, taky mě to napadlo a nemoh jsem se rozhodnout jestli to nechat někde válet nebo založit var